homeassistant/components/zha/cover.py

```diff
@@ -145,13 +162,23 @@ async def async_open_cover(self, **kwargs: Any) -> None:
             raise HomeAssistantError(f"Failed to open cover: {res[1]}")
         self.async_update_state(STATE_OPENING)
 
+    async def async_open_cover_tilt(self, **kwargs: Any) -> None:
+        """Open the cover tilt."""
+        await self._cover_cluster_handler.go_to_tilt_percentage(0)
+
     async def async_close_cover(self, **kwargs: Any) -> None:
         """Close the window cover."""
         res = await self._cover_cluster_handler.down_close()
         if res[1] is not Status.SUCCESS:
             raise HomeAssistantError(f"Failed to close cover: {res[1]}")
         self.async_update_state(STATE_CLOSING)
 
+    async def async_close_cover_tilt(self, **kwargs: Any) -> None:
+        """Close the cover tilt."""
+        res = await self._cover_cluster_handler.go_to_tilt_percentage(100)
+        if not isinstance(res, Exception) and res[1] is Status.SUCCESS:
```

Comment:
This method should mirror `async_close_cover` (e.g. the `isinstance` check can be removed and an exception should be raised if the command fails).